storybook/webpack.config.js

```diff
@@ -0,0 +1,18 @@
+const path = require( 'path' );
+
+module.exports = ( { config } ) => {
+	config.module.rules.push(
+		{
+			test: /\/stories\/.+\.js$/,
+			loaders: [ require.resolve( '@storybook/source-loader' ) ],
+			enforce: 'pre',
+		},
+		{
+			test: /\.scss$/,
```

Comment:
In theory we could avoid this by loading the produced CSS like we do for the components. I feel we should consolidate on one way of loading styles in Storybook.

Reply:
@ItsJonQ - I guess this is a good follow-up to decide how we tackle CSS in Storybook.